src/interactableComponents/drawer/Swipeable.js

```diff
@@ -59,9 +59,9 @@ export default class Swipeable extends Component<PropType, StateType> {
   static defaultProps = {
     friction: 1,
     overshootFriction: 1,
-    // useNativeAnimations: true // issue in iPhone 5
+    useNativeAnimations: true
```

Comment:
Did you test it on iPhone5?

Reply:
for react-native-ui-lib we will enable it for all devices
for our private library i will disable it for iPhone5

Reply:
But the bug exists in the public repo as well. I can't leave it like this

Reply:
@roiberlin keep the default value false here and just expose the prop thru Drawer. 
in the private repo we'll turn it on only if it's not iPhone5

btw, @Inbal-Tish in my opinion, the performance here is preferable on the iPhone5 issue. 
your call

Reply:
I didn't see a hit on performance when I turned it off. We need to find a way to actually measure it

Reply:
Done